.yarn/patches/@web3-onboard-core-npm-2.17.0-158dbf89b9.patch

```diff
@@ -8,12 +8,26 @@ index 06ea0c4a094b86395ce012089b34d6f143cc7101..dbd1c739538b41f4c3a06997e735b111
  			create_component(chainbadge.$$.fragment);
 -			attr(div, "class", "absolute chain-icon-container svelte-jvic9v");
 +      attr(div, "class", "absolute chain-icon-container svelte-jvic9v");
-+      attr(div, "part", "sidebar-heading-img");
```

Comment:
why does it needed ? could it be overridden a different way ? e.g. by manipulating DOM by some event
(like https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) ?

patching packages is a last resort. use it if there are really no other way/workaround please

Reply:
This patch exists already since a long time
The reason we have to patch it is because the web3-onboard module is loaded in a shadow-dom
there is no current way (that I know of) to target CSS within this dom